plugin/evm/orderbook/memory_database.go

```diff
@@ -154,8 +155,15 @@ func (order Order) getExpireAt() *big.Int {
 	return big.NewInt(0)
 }
 
+func (order Order) isPostOnly() bool {
+	if rawOrder, ok := order.RawOrder.(*LimitOrderV2); ok {
```

Comment:
what happens when this is called for other order types like IOC?

Reply:
if `order.RawOrder` is of type IOCOrder, then `ok` will be false